# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ncompress/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some glibc-2.3.4 / gcc-3.4 changes.

  - Rene Rebe <rene@exactcode.de>

--- ncompress-4.2.4/compress42.c.orig	2004-05-03 19:39:12.000000000 +0200
+++ ncompress-4.2.4/compress42.c	2004-05-03 19:42:08.000000000 +0200
@@ -172,7 +172,7 @@
 	extern	void	*malloc	LARGS((int));
 	extern	void	free	LARGS((void *));
 #ifndef _IBMR2
-	extern	int		open	LARGS((char const *,int,...));
+	/* extern	int		open	LARGS((const char *,int,...)); */
 #endif
 	extern	int		close	LARGS((int));
 	extern	int		read	LARGS((int,void *,int));
@@ -638,7 +638,7 @@
 	} ;
 #endif
 
-void  	main			ARGS((int,char **));
+int   	main			ARGS((int,char **));
 void  	Usage			ARGS((void));
 void  	comprexx		ARGS((char **));
 void  	compdir			ARGS((char *));
@@ -691,7 +691,7 @@
  *   deterministic, and can be done on the fly.  Thus, the decompression
  *   procedure needs no input table, but tracks the way the table was built.
  */ 
-void
+int
 main(argc, argv)
 	REG1	int 	 argc;
 	REG2	char	*argv[];
